pkg/kubelet/api/v1alpha1/runtime/api.proto

```diff
@@ -722,6 +722,8 @@ message AttachRequest {
     optional string container_id = 1;
     // Whether to stream stdin.
     optional bool stdin = 2;
+    // Whether use tty
```

Comment:
nit (just for consistency): missing a period at the end. 
s/Whether use/Whether to